.changeset/healthy-actors-hunt.md

```diff
@@ -0,0 +1,6 @@
+---
+"@nomiclabs/hardhat-e2e-tests": patch
```

Comment:
```suggestion
```

We need to remove this or changesets will fail

Reply:
that's right, I always forget that